governance/polls/Initial Parameters for StarkNet Simple Bridge - March 28, 2022.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Initial Parameters for StarkNet Simple Bridge - March 28, 2022
+summary: Signal your support or opposition to the following initial parameters, designed for the deployment of StarkNet Simple Bridge.
+discussion_link: https://forum.makerdao.com/t/starknet-simple-bridge-limit-poll-risk-assessment-mitigation/14158
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-28T16:00:00
+end_date: 2022-03-31T16:00:00
+---
+# Poll:Initial Parameters for StarkNet Simple Bridge - March 28, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Starknet Engineering Core Unit. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March 28 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+The community can vote in this poll to express support or opposition to implementing a simple bridge limit of 5 million DAI. The bridge limit is the maximum amount of DAI that can be in the L1 bridge contract. Hence it is the maximum amount that can be lost by users. This will not include the Fast Withdrawals component which is planned for release later this year and will be put through a separate governance process.
+
+The bridge launch will require the completion of all the mandatory items on the [checklist](https://docs.google.com/spreadsheets/d/1lg4LFQCW7x4bcR8gEgHoE7DB2rqa4GA4DPSUdMOgBsI/edit#gid=718243816).
+Users will use the simple bridge at their own risk. It will be mentioned on the bridge interface.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/starknet-simple-bridge-limit-poll-risk-assessment-mitigation/14158) to help inform your position before voting. For further information on the discussion please visit the previous [thread](https://forum.makerdao.com/t/starknet-dai-bridge-and-starknet-platform-technical-risk-assessment/13950) and [Risk Mitigation Plan](https://docs.google.com/spreadsheets/d/1aByYNfjz1m7o9jyiIhHq-G6OEKq--wu0eoQXzVgc4XE/edit#gid=0). 
```

Comment:
Maybe callout the other thread as the technical assessment? Might be better to structure this like the linked threads when we do collateral onboarding (Bullet points)